chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/rules/developer/DeveloperEditTransparencyAttestationValidation.java

```diff
@@ -0,0 +1,63 @@
+package gov.healthit.chpl.manager.rules.developer;
+
+import java.util.Objects;
+
+import org.ff4j.FF4j;
+
+import gov.healthit.chpl.FeatureList;
+import gov.healthit.chpl.dto.DeveloperACBMapDTO;
+import gov.healthit.chpl.dto.DeveloperDTO;
+import gov.healthit.chpl.manager.rules.ValidationRule;
+import gov.healthit.chpl.permissions.ResourcePermissions;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+
+public class DeveloperEditTransparencyAttestationValidation extends ValidationRule<DeveloperValidationContext> {
+    private FF4j ff4j;
+    private ResourcePermissions resourcePermissions;
+
+    public DeveloperEditTransparencyAttestationValidation(final FF4j ff4j,
+            final ResourcePermissions resourcePermissions) {
+        this.ff4j = ff4j;
+        this.resourcePermissions = resourcePermissions;
+    }
+
+    /**
+     * ROLE_ACB cannot edit Transparency Attestation as per OCD-3164
```

Comment:
I don't like this comment. OCD-3164 is not visible to the public, but our code is, and the comment itself doesn't describe the function accurately, in that the function also depends on a flag state, whereas the comment has no mention of that. I'd prefer just getting rid of it entirely, especially as the function itself is small enough that it's self documenting

Reply:
Remove the comment and pushed change